sqldelight-compiler/src/main/kotlin/app/cash/sqldelight/core/compiler/QueryGenerator.kt

```diff
@@ -192,20 +193,16 @@ abstract class QueryGenerator(
 
     // Adds the actual SqlPreparedStatement:
     // statement = database.prepareStatement("SELECT * FROM test")
-    val executeMethod = if (query is NamedQuery &&
+    val isNamedQuery = query is NamedQuery &&
```

Comment:
this stuff is more complex on sql-psi-dev but we can merge this and rebase that

Reply:
I have some spare time today to look into rebasing sql-psi-dev if you haven't started already